src/components/actionPayloadRenderers/SessionPayloadRendererContainer.tsx

```diff
@@ -26,6 +26,7 @@ export default class Component extends React.Component<Props, {}> {
 
         return (
             <TextPayloadRenderer
+                classNameEx="cl-ux-bolditalic"
                 original={renderStringUsingEntityNames}
```

Comment:
Rather than pollute the TextPayloadRenderer, why not create a "SessionPayloadRenderer" akin to the "ApiPayloadRenderer" and "CardPayloadRenderer".  This give us more flexibility as well as make it easy to support the multi-line thing

Reply:
I considered this idea, then figured we might want to enhance more text rendering down the road. If true, extending the props on the text rendering opens styling up to everyone, not just session rendering